Tetris Clone C++/GridController.hpp

```diff
@@ -7,12 +7,15 @@ extern const bool debug;
 extern const int pixels;
 extern const int rows;
 extern const int columns;
+
 class TetrisPiece;
-class GridController
-{
+
+// typedef enum TETRONIMO_TYPES TetronimoType;
+
+class GridController {
 private:
-    TetrisPiece *pieces;
-    bool grid[10][22];
+    // TetrisPiece *pieces;
+    TetronimoType grid[22][10];
```

Comment:
I'd really like to reach a point where we can define rows and columns in one place, and not have to type 10,22 everywhere. we should discuss